src/main/java/gov/usgs/owi/nldi/services/ConfigurationService.java

```diff
@@ -35,6 +35,10 @@ public String getRootUrl() {
 		return getDisplayProtocol() + "://" + getDisplayHost() + getDisplayPath() + "/linked-data";
 	}
 
+	public String getSwaggerUrl() {
```

Comment:
I think there may be less confusion in the long run if getRootUrl() where renamed getLinkedDataUrl() and this method (getSwaggerUrl()) would be renamed getRootUrl().